README.md

```diff
@@ -78,6 +78,37 @@ Other branches are deployed to https://models-resources.concord.org/question-int
 
 You can view the status of all the branch deploys [here](https://travis-ci.org/concord-consortium/question-interactives/branches).
 
+To deploy a production release:
+
+1. Copy CHANGELOG.md to CHANGES.md, and add a list of PT stories related to the release into temporary CHANGES.md
+    - Update the version and date at the top of the CHANGES.md
+    - Run `git log --reverse v<last version>...HEAD | grep '#'` to see a list of PR merges and stories that include PT ids in their message
+    - In a PT workspace that includes Orange and Teal boards, search for the `label:"question-interactives-<new version>" includedone:true`. You can select all, and export as CSV. Then copy the id and title columns.
```

Comment:
I don't know if you want to standardize on this, but I've been using my script to do this step which is easier, than the CSV export:
https://github.com/concord-consortium/dev-templates/tree/main/scripts

Reply:
Done, I've updated step 1 to make use of the dev-templates script.